controllers/authController.js

```diff
@@ -69,3 +72,27 @@ exports.register = async (req, res) => {
     })
   }
 }
+
+async function sendVerifMail(mail) {
+    let transporter = nodemailer.createTransport({
+      service: "Gmail",
+      auth: {
+        user: 'testdsc242@gmail.com',
+        pass: 'testdsc0'
+      }
+    });
+
+    let mailOption = {
+      from: 'Test Test <testdsc242@gmail.com>', // sender address
+      to: 'rusudaniel5799@gmail.com', // list of receivers
+      subject: "Hello", // Subject line
```

Comment:
let's add in our config file the subject and from.
`to` should be the email from the user.
please add the template with a replace variable ability.

Reply:
How can I use template1.html as variable to mail body?

Reply:
@RDaniel99 citesti templatet-ul si dupa inlocuiesti ce e de inlocuit si dupa il pui la html